src/decision/include/LidarDecision.h

```diff
@@ -48,13 +48,27 @@ class LidarObstacle {
     LidarObstacle(std::vector<Reading> readings);
 
     /**
+    * Gets the distance of the first data from a Obstacle group
+    *
+    * @return the distance of the first data from a Obstacle group
+    */
+    distance_t getFirstDistance();
```

Comment:
Since the vector of distances is sorted, this function is actually returning the **minimum** distance to the obstacle. As such, `getMinDistance` might be a better function name.